contentcuration/contentcuration/models.py

```diff
@@ -120,6 +120,15 @@ def generate_file_on_disk_name(checksum, filename):
         os.makedirs(directory)
     return os.path.join(directory, h + ext.lower())
 
+def generate_storage_path(checksum, filename):
```

Comment:
This should be something like `generate_storage_url`, as otherwise it sounds like a filepath.